frontend/src/components/Form.jsx

```diff
@@ -37,80 +37,17 @@ function Form() {
       transitionLeave={true}
     >
       <container className="formContainer">
-        <form className="form" onSubmit={e => handleSubmit(e)}>
-          {/* LastName -------------------------------------------------------------------------------------------- */}
+        <form
+          className="form"
+          onSubmit={e => {
```

Comment:
(`handleSubmit` should be called from here, it'd prevent you from firing another event on the validation button click. Once again, it's not blocking and only a minor performance matter, I'll let it pass for now)